### PR TITLE
WebSockets Next: document WebSocket#inboundProcessingMode()

### LINKED ISSUE #41393: WebSockets Next: document `WebSocket#inboundProcessingMode()`
### Description

https://github.com/quarkusio/quarkus/blob/main/extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/WebSocket.java#L50-L53

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by mkouba**
- resolves #41393

### CODE REVIEW

File: extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java [hunk 1]
```diff
@@ -12,12 +14,12 @@
 public enum InboundProcessingMode {
 
     /**
-     * Messages are processed serially, ordering is guaranteed.
+     * Events are processed serially, ordering is guaranteed.
```
Comment (geoand) [inline extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java:17]: Should we also describe what an event is here?

Comment (geoand) [reply extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java:17]: Or maybe just link to the Javadoc of the class

Comment (mkouba) [reply extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java:17]: Link to the javadoc of the class does not make sense IMO.

Comment (geoand) [reply extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/InboundProcessingMode.java:17]: kk

**Review by geoand — APPROVED** [review]

**Pull request merged**